src/transaction/locator_sr.c

```diff
@@ -5329,8 +5333,12 @@ locator_update_force (THREAD_ENTRY * thread_p, HFID * hfid, OID * class_oid, OID
   REPL_INFO repl_info;
   TDE_ALGORITHM tde_algo = TDE_ALGORITHM_NONE;
 
+  OID superclass_oid;
+
   assert (class_oid != NULL && !OID_ISNULL (class_oid));
 
+  OID_SET_NULL (&superclass_oid);
```

Comment:
For information only
the following is the same.
```
OID superclass_oid = OID_INITIALIZER;
```

Reply:
Ok, thanks.